src/components/file-browser/DownloadFileLink.ts

```diff
@@ -22,7 +22,7 @@ export const DownloadFileLink = (props: DownloadFileLinkProps) => {
       ...Utils.newTabLinkProps,
       disabled: !downloadUrl,
       download: basename(file.path),
-      href: downloadUrl,
+      href: downloadUrl || undefined,
```

Comment:
`downloadUrl` is `string | null`, but Clickable wants `string | undefined`.

Reply:
This could also be a useful inline comment